pcl_ros/src/pcl_ros/filters/filter.cpp

```diff
@@ -176,10 +176,24 @@ pcl_ros::Filter::unsubscribe()
 pcl_ros::Filter::Filter(std::string node_name, const rclcpp::NodeOptions & options)
 : PCLNode(node_name, options)
 {
-  pub_output_ = create_publisher<PointCloud2>("output", max_queue_size_);
   RCLCPP_DEBUG(this->get_logger(), "Node successfully created.");
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////////
+void
+pcl_ros::Filter::createPublishers()
+{
+  auto pub_options = rclcpp::PublisherOptions();
+  pub_options.event_callbacks.matched_callback = [this](rclcpp::MatchedInfo & /*info*/) {
+      if (pub_output_->get_subscription_count() == 0) {
+        unsubscribe();
+      } else {
+        subscribe();
```

Comment:
See jazzy PR branch

Reply:
I checked the PR for the jazzy branch, but it seems there are no reviews. Could it still be pending?

Reply:
Sigh. GitHub must have blipped. Its up :-)